uint64 timestamp	                # time since system start (microseconds)

float64 lat			                # Latitude, (degrees)
float64 lon			                # Longitude, (degrees)
float32 alt			                # Altitude AMSL, (meters)
float32 alt_ellipsoid		        # Altitude above ellipsoid, (meters)